content/sensu-core/2.0/getting-started/installation-and-configuration.md

```diff
@@ -68,30 +70,32 @@ Copy the example backend config file to the default config path.
 sudo cp /etc/sensu/backend.yml.example /etc/sensu/backend.yml
 {{< /highlight >}}
 
-The backend config requires `state-dir` to be set. The example config sets `state-dir` to `/var/lib/sensu` by
-default.
+The backend config requires `state-dir` to be set.
+The example config sets `state-dir` to `/var/lib/sensu` by default.
 
 ### 3. Start the service
 
-Start the services using systemd.
+Start the backend using systemd.
 
 {{< highlight shell >}}
 sudo systemctl start sensu-backend
 {{< /highlight >}}
 
-You can verify that the services are properly running using the log files.
+You can verify that the backend is running properly using the log files.
```

Comment:
```suggestion
You can verify that sensu-backend is running properly using the log files.
```

Reply:
I'm thinking it may be better to say the service that we're verifying instead of just saying "agent"